src/ngraph/op/softmax.cpp

```diff
@@ -90,3 +97,70 @@ void op::Softmax::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVect
     auto x = input_value(0);
     adjoints.add_delta(x, adjoint);
 }
+
+// *** SOFTMAX OP SET V1 ***
+const string op::v1::Softmax::type_name{"Softmax"};
```

Comment:
Same `type_name` value for both versions ? We will possibly do something like  `op->type_name == op::Softmax::v0::type_name` to check for v0. Now we have to add a check for op version `&& op->get_op_version() == 0`

Reply:
See #3533